.github/CONTRIBUTING.md

```diff
@@ -1,20 +1,22 @@
-# differentialabundance/differentialabundance: Contributing Guidelines
+# nf-core/differentialabundance: Contributing Guidelines
```

Comment:
It's a bit weird that here everywhere was `differentialabundance`, also in the template branch so either everything would need to be created again from scratch with the `nf-core create` command, or at least just the template branch.